bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/ewkwmlvjj_5f_LO/ewkwmlvjj_5f_LO_proc_card.dat

```diff
@@ -0,0 +1,13 @@
+import model sm-ckm_no_b_mass
+#import model sm-ckm_lepton_masses_no_b_mass
+#switch to diagonal ckm matrix if needed for speed
```

Comment:
please apply these requirements (in particular commented lines) https://twiki.cern.ch/twiki/bin/viewauth/CMS/Moriond18MC#Checks_to_the_datacards_BEFORE_T

Reply:
Hi @perrozzi , I had missed this twiki. Now that I am checking it, I see in the run card we need to set T = pdfwgt. Please confirm if it is applicable here also.